dev-db/etcd-starter/etcd-starter-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2014 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+# $Header:$
+#
+
+EAPI=5
+CROS_WORKON_PROJECT="coreos/etcd-starter"
+CROS_WORKON_LOCALNAME="etcd-starter"
+CROS_WORKON_REPO="git://github.com"
+COREOS_GO_PACKAGE="github.com/coreos/etcd-starter"
+inherit coreos-go cros-workon
+
+if [[ "${PV}" == 9999 ]]; then
+    KEYWORDS="~amd64"
+else
+    CROS_WORKON_COMMIT="e6265b3a028225a8b57ddbfc3ad1992abc89974f" # v0.0.2
+    KEYWORDS="amd64"
+fi
+
+DESCRIPTION="etcd-starter"
+HOMEPAGE="https://github.com/coreos/etcd-starter"
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+DEPEND=">=dev-lang/go-1.2"
+
```

Comment:
needs something like this:
```sh
RDEPEND="!dev-db/etcd:2[etcd_protocols_1]
    !dev-db/etcd:1[etcd_protocols_2]"
```

Basically this ebuild installs /usr/bin/etcd so it needs to conflict with the situations in which the other two ebuilds may install /usr/bin/etcd. Alternatively (if etcd-starter is ready) always install etcd-starter as /usr/bin/etcd so we can drop all this weird use flag logic to determine who installs /usr/bin/etcd